### PR TITLE
Upgrade scalardl to 3.0.1 and schema-loader-image tag to 3.0.0

### PR DESCRIPTION
**Pull request by tei-k**

### CODE REVIEW

**Review by feeblefakie — COMMENTED** [review]
Thank you!
Left 1 comment.

I think we should tag releases. Can you do it after it is merged?
It would be great if you can do it for old versions as well.

File: charts/scalardl/Chart.yaml [hunk 1]
```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: scalardl
 description: Implementation scalardl.
 type: application
-version: 1.2.3
-appVersion: 2.1.0
+version: 1.3.0
```
[the remark this answers is not in the document]
Comment (tei-k) [reply charts/scalardl/Chart.yaml]: OK! Upgraded both `ledger` and `schema-loader` to 2.0.0 in 8d8f289.

**Comment by feeblefakie** [comment]
@tei-k CI seems failed. Can you fix it?

**Comment by tei-k** [comment]
Worked fine in my local & AKS.
```
$ k get po -A
NAMESPACE     NAME                                          READY   STATUS    RESTARTS   AGE
default       cassandra-0                                   1/1     Running   0          3m38s
default       scalardl-10flnb8gsv-envoy-cdfd56fd4-9b48d     1/1     Running   0          86s
default       scalardl-10flnb8gsv-envoy-cdfd56fd4-lfxmz     1/1     Running   0          86s
default       scalardl-10flnb8gsv-envoy-cdfd56fd4-x9kxh     1/1     Running   0          86s
default       scalardl-10flnb8gsv-ledger-6cd9dc4bf8-96nq7   1/1     Running   1          86s
default       scalardl-10flnb8gsv-ledger-6cd9dc4bf8-lrqbr   1/1     Running   1          86s
default       scalardl-10flnb8gsv-ledger-6cd9dc4bf8-tn5wg   1/1     Running   1          86s
kube-system   coredns-74ff55c5b-6plfq                       1/1     Running   0          4m46s
kube-system   etcd-minikube                                 1/1     Running   0          4m57s
kube-system   kube-apiserver-minikube                       1/1     Running   0          4m57s
kube-system   kube-controller-manager-minikube              1/1     Running   0          4m57s
kube-system   kube-proxy-6c4nw                              1/1     Running   0          4m46s
kube-system   kube-scheduler-minikube                       1/1     Running   0          4m57s
kube-system   storage-provisioner                           1/1     Running   1          4m58s
```

Maybe this is a pull limit issue of dockerhub's cassandra. (or slow start issue) 🤔 
```
statefulset.apps/cassandra created
service/cassandra created
Wait until cassandra is running
NAME              READY   STATUS    RESTARTS   AGE    IP       NODE     NOMINATED NODE   READINESS GATES
pod/cassandra-0   0/1     Pending   0          100s   <none>   <none>   <none>           <none>

NAME                 TYPE        CLUSTER-IP   EXTERNAL-IP   PORT(S)    AGE     SELECTOR
service/cassandra    ClusterIP   None         <none>        9042/TCP   100s    app=cassandra
service/kubernetes   ClusterIP   10.96.0.1    <none>        443/TCP    3m15s   <none>

NAME                   ENDPOINTS         AGE
endpoints/cassandra    <none>            100s
endpoints/kubernetes   172.18.0.2:6443   3m15s

NAME                               STATUS     ROLES    AGE     VERSION    INTERNAL-IP   EXTERNAL-IP   OS-IMAGE       KERNEL-VERSION     CONTAINER-RUNTIME
node/chart-testing-control-plane   NotReady   master   3m17s   v1.18.15   172.18.0.2    <none>        Ubuntu 20.10   5.8.0-1036-azure   containerd://1.4.0-106-gce4439a8
node/chart-testing-worker          NotReady   <none>   2m41s   v1.18.15   172.18.0.3    <none>        Ubuntu 20.10   5.8.0-1036-azure   containerd://1.4.0-106-gce4439a8
```

**Comment by tei-k** [comment]
Nodes is not ready.
```
NAME                               STATUS     ROLES    AGE     VERSION   INTERNAL-IP   EXTERNAL-IP   OS-IMAGE       KERNEL-VERSION     CONTAINER-RUNTIME
node/chart-testing-control-plane   NotReady   master   3m17s   v1.19.7   172.18.0.2    <none>        Ubuntu 20.10   5.8.0-1036-azure   containerd://1.4.0-106-gce4439a8
node/chart-testing-worker          NotReady   <none>   2m42s   v1.19.7   172.18.0.3    <none>        Ubuntu 20.10   5.8.0-1036-azure   containerd://1.4.0-106-gce4439a8

```

**Comment by tei-k** [comment]
>@tei-k CI seems failed. Can you fix it?

@feeblefakie I fixed the ci failure! Please check it again.

**Comment by tei-k** [comment]
>I think we should tag releases. Can you do it after it is merged?

OK!

>It would be great if you can do it for old versions as well.

I need the scope of old versions. Do you mean just the `schema-loader` version 1.3.1?

**Review by superbrothers — APPROVED** [review]
:+1:

**Review by ymorimo — APPROVED** [review]
👍

**Review by feeblefakie — APPROVED** [review]
LGTM!

**Pull request merged**